README.md

```diff
@@ -15,6 +19,9 @@
 1. Start Mongo: `mongod`
 2. Connect to the database: `node app/server.js`
 
+# Run Tests
+1. `npm test`
```

Comment:
Both run the tests, tbh. But `npm test` is 1 less character than `gulp test` lol. I don't have a real reason, I could add both or `gulp` its really preference here.